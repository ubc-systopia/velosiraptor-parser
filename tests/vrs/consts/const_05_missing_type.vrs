const FOO = 1234;
