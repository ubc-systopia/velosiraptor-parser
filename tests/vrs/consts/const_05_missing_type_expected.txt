error: expected `:`, found `=`
     --> tests/vrs/consts/const_05_missing_type.vrs:1:11
      |
    1 |  const FOO = 1234;
      |            ^ help: add `:` here.
